.github/workflows/Lockfile.yml

```diff
@@ -12,9 +12,18 @@ jobs:
       contents: write
     runs-on: ubuntu-latest
     steps:
-      - uses: step-security/harden-runner@91182cccc01eb5e619899d80e4e971d6181294a7 # v2.10.1
+      - name: Harden Runner
+        uses: step-security/harden-runner@91182cccc01eb5e619899d80e4e971d6181294a7 # v2.10.1
         with:
-          egress-policy: audit
+          disable-sudo: true
+          egress-policy: block
```

Comment:
What the possible options? I could only see `block` and `audit`.

Reply:
Those are the only possible values:
https://github.com/step-security/harden-runner/blob/main/action.yml

If it's set to block you can choose to not report the results to their API.